source/publishing_data/07_configuring_visualizations/04_configuring_image_view/image.rst

```diff
@@ -1,18 +1,96 @@
-Configuring the image gallery
-=============================
+Configuring the Images visualization
+====================================
 
-.. admonition:: Note
-   :class: note
 
-   The images view is only available when there is at least a *file* field, and if at least one file contained in this field is an image, or a document that can generate a thumbnail.
+.. admonition:: Prerequisite
+   :class: important
 
+   For the Images visualization to be available, the dataset must contain at least one :ref:`file field<choosingtype>`, containing at least one image or any other media type file that can generate a thumbnail.
```

Comment:
> or any other media type file that can generate a thumbnail.

@thoumasd @remilejeune 
Are we still generating thumbnails for PDF and things like that?
If so, it could be nice to explicitely indicate which files can generate these thumbnails (are there more than PDF?)

Reply:
We only generate thumbnails for images. And indeed, we need to cleary state the images types for which we generate thumbnails: "gif", "png", "jpeg", "jpg", "tif", "tiff", "bmp".